crypto/fipsmodule/modes/cbc.c

```diff
@@ -85,12 +85,10 @@ void CRYPTO_cbc128_encrypt(const uint8_t *in, uint8_t *out, size_t len,
     }
     (*block)(out, out, key);
     iv = out;
+    // This will always be true as a result of the first while loop
```

Comment:
Then we should instead change the `while (len)` above to an `if (len > 0)` and then there's no reason to `break`.